Diama1/EPIC-Mail
# Ft delete email 164400093

**What does this PR do?**

This PR will enable the user to delete an email.

**Description of Task to be completed?**

the user will be able to specify the message to be deleted then after he will delete it from inbox or among sent messages. 



**How should this be manually tested?**

Clone the branch
Install all dependencies using npm install
Run the server by running this command (npm start) then go to your postman choose DELETE and type api/v1/messages/message_id
Any background context you want to provide?
N/A
What are the relevant pivotal tracker stories?
User should be able to delete an email. 164400093
Screenshots (if appropriate)
N/A

Questions:
N/A
